executor/prepared.go

```diff
@@ -207,8 +207,6 @@ func (e *ExecuteExec) Build() error {
 	var err error
 	if IsPointGetWithPKOrUniqueKeyByAutoCommit(e.ctx, e.plan) {
 		err = e.ctx.InitTxnWithStartTS(math.MaxUint64)
-	} else {
-		err = e.ctx.ActivePendingTxn()
```

Comment:
Is it missed in the last PR?

Reply:
I think, yes, Prepared Executor is forgot
https://github.com/pingcap/tidb/pull/8260/files#diff-350127760839dbfd52d23927f7ff2d95L290